Exoft.Gamification.Api.Services.Interfaces/Repositories/IRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Exoft.Gamification.Api.Data.Core.Entities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exoft.Gamification.Api.Services.Interfaces.Repositories
+{
+    public interface IRepository<T> where T: Entity
+    {
+        Task AddAsync(T entity);
```

Comment:
Why dont just add here GetById async? Then we will avoid GetAchievementAsync-like methods